Userland/DevTools/UserspaceEmulator/CMakeLists.txt

```diff
@@ -13,4 +13,4 @@ set(SOURCES
 )
```

Comment:
Typos in commit message:
singlestep -> single-step
exectuion -> execution
scafold -> scaffold